src/components/Data/Collections/Create.vue

```diff
@@ -52,21 +99,25 @@
 
   export default {
     name: 'CollectionCreate',
-    props: {
-      index: String
-    },
     components: {
       Headline,
       JsonEditor
     },
+    props: {
+      index: String
+    },
     data () {
       return {
         collectionName: null,
+        settingsOpen: false,
         error: '',
         isRealTime: false
       }
     },
     methods: {
+      cancel () {
+        window.history.back()
```

Comment:
I'm not very comfortable with `history.back()`.
If the user copy/past the url and comes from an other site, he will be redirect on this other website, weird na?